logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/usecase/AnswerCallUseCase.kt

```diff
@@ -42,6 +52,21 @@ internal class AnswerCallUseCaseImpl(
     override suspend fun invoke(
         conversationId: ConversationId
     ) {
+        // mute or un-mute call when answering/joining
+        allCalls().map {
```

Comment:
not sure if i understand the fix, but this code is executed before answering the call, so allCalls will probably emit and empty list 😅

Reply:
We were muting the call in [SharedCallingViewModel](https://github.com/wireapp/wire-android-reloaded/pull/1798/files#diff-611a24f28e38a5ffd94a72c4355367994ad0c86a934031f0b331f18aacbc5a5d) when the call is established, I changed it to be done when answering

 and that's fine if it's before, we already have an incoming call so It will not emit an empty list